components/movementsensor/gpsrtkserial/gpsrtkserial.go

```diff
@@ -501,29 +498,18 @@ func (g *rtkSerial) receiveAndWriteSerial() {
 					return
 				}
 
-				g.logger.Debug("No message... reconnecting to stream...")
 				if g.isVirtualBase {
 					g.logger.Debug("reconnecting to the Virtual Reference Station")
-					err = g.sendGGAMessage()
 
+					err = g.getNtripFromVRS()
 					if err != nil && !errors.Is(err, io.EOF) {
 						g.err.Set(err)
 					}
```

Comment:
If we had this error, we used to try one more time, and if we still had errors, we'd return early. Now that we're not trying one more time, should we return early here?